test/cmake_build.bat

```diff
@@ -0,0 +1,2 @@
+REM for detailed log you can add -- /verbosity:detailed
+cmake --build ./build
```

Comment:
@Xrayez  @Calinou  , I think I should remove these .bat files, as I do not see any shell scripts or bat files. On the other hand this is a sample project so it makes me doubt a bit.

Reply:
I don't know to be honest, I think it's better to leave this up for another PR.

Reply:
I will remove it

Reply:
I will create another PR to open the discussion about those .bat / shell scripts